docs/userguide/conversion/compression.md

```diff
@@ -18,7 +18,7 @@ Unicode (usually in the Basic Latin range).
 For additional detail about the compression algorithm, which has been approved
 by the Unicode Consortium, please refer to [Unicode Technical Report #6 (A
 Standard Compression Scheme for
-Unicode)](http://www.unicode.org/unicode/reports/tr6/) .
+Unicode)](http://www.unicode.org/unicode/reports/tr6/).
```

Comment:
minor nit: I'd suggest using https instead of http, but you don't have to fix/change it. :)